src/core/Akka/Akka.csproj

```diff
@@ -53,7 +53,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
+    <DefineConstants>$(DefineConstants);CORECLR;AKKAIO</DefineConstants>
```

Comment:
Both NodeTestRunner and MultiNodeTestRunner uses the Tcp class in each Program.cs